soccer/src/soccer/strategy/agent/position/seeker.cpp

```diff
@@ -182,7 +187,7 @@ double Seeker::eval_point(rj_geometry::Point ball_pos, rj_geometry::Point curren
 
     // Additional heuristics for calculating optimal point
     double ball_proximity_loss = (current_point - ball_pos).mag() * .002;
-    double goal_distance_loss = (9.0 - current_point.y()) * .008;
+    double goal_distance_loss = (8.5 - current_point.y()) * 0.15 + abs(current_point.x()) * 0.15;
```

Comment:
magic numbers ??

Reply:
numbers like `8.5`, which seems to be the measurement of something on the field, usually break when we play on a different size field. note that this happens a lot, because our shop field is half the size of our sim/comp field. can we change this to be derived from field dimensions? even if there's a "magic" offset from midfield or something, better than nothing.

Reply:
other constants like weights need to be named and the reasoning for their value needs to be commented, unless their purpose is really obvious. an easy trick is to make them all `static constepxr`s defined at the top of the function or in the header

Reply:
Good points. We can merge for now and I can fix when we fix the seeking oscillation if that's good with you.

Reply:
sounds good